lib/collection.js

```diff
@@ -324,7 +338,7 @@ Collection = Events.inherit(/** @lends Collection.prototype*/{
         } else {
             return new this.modelType(model, {
                 collection: this,
-                isNested: true
+                owner: this._owner
```

Comment:
it is now, in case collection doesn't owned by parent model
this._owner = options.owner || this;